platform/core/src/utils/cornerstoneWADOImageLoader.js

```diff
@@ -0,0 +1,5 @@
+export function getCornerstoneWADOImageLoader() {
+  return import(
+    /* webpackChunkName: "CornerstoneWADOImageLoader" */ 'cornerstone-wado-image-loader'
+  );
+}
```

Comment:
Hmm.. This is tricky. Maybe instead of having `cornerstone-wado-image-loader` as a dependency in `@ohif/core`, we update `dicomLoaderService.js`, studyInstanceHelpers.js`, and `studyLoadingListener.js` to take an image loader in their constructors. Then we don't need to know when/if it has already been initialized?

Reply:
Passing it through constructor will require changes in multiple places across other packages, at least for `DicomLoaderService`. 

One not-so-clean alternative is to use to from global as `initCornerstoneWADOImageLoader` is setting up `window.cornerstoneWADOImageLoader`.